docs/DeveloperGuide.md

```diff
@@ -286,20 +286,32 @@ Use case ends.
 
 3a. The given index is invalid
 
-> 3a1. AddressBook shows an error message <br>
+> 3a1. Task Manager shows an error message <br>
   Use case resumes at step 2
 
-{More to be added}
+#### Use case: Search task
+
+**MSS**
+
+1. User type the correct command in the command box(search KEYWORD)
```

Comment:
There is no need to be too specific about the syntax of the search command. It is assumed to be correct. Any invalid cases will be stated in the extensions. Perhaps you could rephrase this to follow the writing style of the previous use cases above. i.e. `1. User requests to list persons`